setup.py

```diff
@@ -59,5 +59,9 @@
         "Documentation": "https://qiskit.org/documentation/",
         "Source Code": "https://github.com/Qiskit/red-queen",
     },
+    entry_points="""
+        [console_scripts]
+        red-queen=cli:main
+    """,
```

Comment:
This syntax isn't correct, it should be something like:

```suggestion
    entry_points={
        'console_scripts': [
            "red-queen = red_queen.cli:main",
        ]
    }
```